pom.xml

```diff
@@ -25,7 +25,7 @@
     <parent>
         <groupId>org.apache.sling</groupId>
         <artifactId>sling-bundle-parent</artifactId>
-        <version>44</version>
+        <version>52</version>
```

Comment:
Can you update to latest? There are hints on what you need to consider in https://cwiki.apache.org/confluence/x/SI75E

Reply:
The latest parent only builds with Java 17 or later. Is that OK? Aren't there any clients that need to run on JDK 11?

Reply:
Oh, after reading the linked page I can see that it still targets JDK 11, but requires JDK 17+ for the build process. I'll have a try at it.